src/planscape/collaboration/urls.py

```diff
@@ -13,7 +13,12 @@
     ),
     path(
         "invitations/<ctype:target_entity>/<int:object_pk>",
-        GetInvitationsForObject.as_view(),
+        InvitationsForObject.as_view(),
         name="get_invitations",
     ),
+    path(
+        "invitations/<ctype:target_entity>/<int:object_pk>/<int:invite_id>",
+        InvitationsForObject.as_view(),
+        name="update_invitation",
```

Comment:
Let me know if you have thoughts on what url format seems less of a pain here. 

I initially hardcoded the url just to handle role changes for existing userobjectrole records, with the format `update_userobjectrole/<int:userobjectrole_id>`, which is maybe easier to deal with, just less "RESTy" compared to the code above.

Since the userobjectrole already knows what model & object_pk we're dealing with, I don't think it's necessary to pass all of these other path params.

Reply:
so the url would be something like PUT "invitations/planningarea/$planning_area_id/$invitation_id" ? what about the role? would that be part of the post/put body?

Reply:
yep, those would be in the request body in either case  Examples are in the test payloads above, e.g.,
`{"role":"Collaborator"}`